test/test_web.py

```diff
@@ -1428,6 +1428,27 @@ def test_05_test_signout_json(self):
         err_msg = "success message missing"
         assert data['status'] == 'success', err_msg
 
+    @with_context
+    @with_context_settings(ENABLE_TWO_FACTOR_AUTH=True)
+    def test_06_test_disabled_user_signin_1(self):
+        res = self.register()
+        res = self.signout()
```

Comment:
is this admin signin to update user's state?

Reply:
it's just to create the user